SKSpriteButton/Classes/SKSpriteButtonEventListener.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Created by Nader Eloshaiker on 25/3/18.
+// Copyright (c) 2018 Rotate Gears. All rights reserved.
+//
+
+import SpriteKit
+
+public class SKSpriteButtonEventListener {
```

Comment:
I can see why you want to add  this class. 
But this library should be kept as simple as possible. For now, the less public APIs exposed the better.